docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -194,3 +198,35 @@ pytest /tmp/tests/test_mock_tests.py -vv --junitxml=${SHARED_DIR}/junit_output.x
 ```
 
 The shell script above utilizes `pytest` to execute the tests that were copied into the container in the [Creating Containerized Tests](#creating-containerized-tests) section of this guide. The JUnit results will be placed in the `SHARED_DIR` where they can be reported in later steps or other containers.
+
+### Miscellaneous Container Tutorials
+
+#### Copying Files From One Image to Another
+
+In some scenarios, you may need to use multiple images and you may need to copy files from one image to another. For example, the OADP scenario requires multiple private repositories. Rather than cloning the repositories (which would include maintaining a service account and personal access token) into one image, we have decided to [promote an image for each repository](../Scenario_Development/Scenario_Development_Guide.md#multiple-private-repo-environment-images) and copy the files we need into one image.
+
+1. Define base images in your configuration file
+   - For this example, we will only define the `cli` image, but you can define as many as you'd like.
+
+```yaml
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+```
+
+2. Define an image that copies a file from one of your base images
+   - This example copies the oc client into the current directory of my new image
+
+```yaml
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  inputs:
+    cli: # Base image to copy files from
+      paths:
+      - destination_dir: . # Destination of file being copied in the new image
+        source_path: /usr/bin/oc # File to copy from base image
+  to: some-image-name
```

Comment:
Is this the image being created to store the copied files?
Or it can be any existing image where we want to have those files?